tools/testbench/testbench.c

```diff
@@ -522,15 +528,17 @@ static int test_pipeline_load(struct testbench_prm *tp, struct tplg_context *ctx
 }
 
 static void test_pipeline_stats(struct testbench_prm *tp,
-				struct tplg_context *ctx, uint64_t delta)
+				struct tplg_context *ctx, long long delta_t)
```

Comment:
Did this _really_ need changing?

Reply:
I don't remember, I first did a minimum effort then per reviewers' request I've been changing the code. Please let me know if this needs to be changed back.

Reply:
Fair enough!